Design.md

```diff
@@ -3,6 +3,26 @@
 This document captures the design decisions made after discussion in issues. When a design issue is closed, the
 conclusion should be summarized here with a link to the issue.
 
+## OSTree Delivery Format
+
+- Originally discussed in issue [#23](https://github.com/coreos/fedora-coreos-tracker/issues/23). 
+
+### Summary
+
+RPM-OStree supports three delivery models for delivering content to
+end user systems.
+
+- rojig: uses a special rojig RPM and re-assembles OSTree commit from RPMs
+  already on mirrors.
+- OCI: OSTree commits are packaged up in OCI container images and delivered 
+  via a container registry
```

Comment:
Well... RPM-OSTree doesn't technically support that today, right? Specific platforms can make use of https://github.com/openshift/pivot for now, though there's definitely work to be done before one can do e.g. `rpm-ostree rebase oci://....`.

Reply:
yeah that is true. I was just trying to give a brief rundown of the options at a high level. suggested edit?

Reply:
Gotcha. In that case maybe just changing the "RPM-OSTree supports three..." to something like: "There are three proposed delivery models..." ?

Reply:
done!